src/sbvr-api/user.sbvr

```diff
@@ -53,6 +53,9 @@ Fact type:  api key has key
 	Necessity: each key is of exactly one api key
 Fact type:  api key has role
 	Note: An 'api key' will inherit all the 'permissions' that the 'role' has.
+	Term Form: api key role
+	Fact type: api key role has expiry date
+		Necessity: Each user role has at most one expiry date.
```

Comment:
The rule: `Necessity: Each user role has at most one expiry date. ` already exists:

```
Term: user
...
Fact type: user has role
	Term Form: user role
	Fact type: user role has expiry date
		Necessity: Each user role has at most one expiry date.
```	
```suggestion
		Necessity: Each api key role has at most one expiry date.
```

Reply:
👍